content/blog/configure-isut-and-ams-for-vlcm-based-firmware-updates-on-gen-12-servers.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to Configure iSUT and AMS for vLCM-Based Firmware Updates on HPE
+  Gen12 Servers (High Security Mode)
+date: 2025-10-01T02:30:17.312Z
+author: Vijayakannan M
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
+  - OV4VC
+  - VLCM
+  - COM4VC
+  - HPE OneView
+---
+<style> li { font-size: 27px; line-height: 33px; max-width: none; } </style>
+
+HPE Gen12 servers introduce enhanced security by supporting only High Security modes (SecureStandard, CNSA, FIPS). This impacts how you configure **iSUT** (Intelligent System Update Tool) and **AMS** (Agentless Management Service) for **vSphere Lifecycle Manager (vLCM)** based firmware updates. Unlike previous generations, configuration through the **HPE OneView for VMware vCenter** (OV4VC) and **HPE Compute Ops Management plug-in for VMware vCenter** (COM4VC) vLCM Pre-Check page is not available in these modes, as iLO credentials are now required. Instead, you must manually configure AMS and iSUT by creating an application account and providing valid HPE iLO credentials.
+
```

Comment:
Add a line space here and I would add a short sentence to grab the reader's attention to your post.

For example:
In this blog post I'll show you how to....

or

In this blog post, I'd like to share with you a method to...